components/portal/widgets/partials/external-message.html

```diff
@@ -0,0 +1,38 @@
+<!--
+
+    Licensed to Apereo under one or more contributor license
+    agreements. See the NOTICE file distributed with this work
+    for additional information regarding copyright ownership.
+    Apereo licenses this file to you under the Apache License,
+    Version 2.0 (the "License"); you may not use this file
+    except in compliance with the License.  You may obtain a
+    copy of the License at the following location:
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+<div class="overlay__widget-mode" 
+  ng-hide="message_acknowledged || !widget.externalMessageText.trim()">
+  <div class="overlay-content">
+    <p>{{ widget.externalMessageText | limitTo:225}}</p>
+    <md-button ng-if="widget.externalMessageLearnMoreUrl" 
+      class="md-accent md-raised" id="overlay-learn-more-button" 
+      ng-href={{widget.externalMessageLearnMoreUrl}} aria-label=
+      "Learn more about the important message concerning {{widget.fname}}"
```

Comment:
Using `fname` in `aria-label` isn't ideal. Maybe the external message source should be able to and encouraged to provide an `aria-label`? Maybe this should dereference the `fname` to the portlet title?

Reply:
Or use `widget.title` which should be available.

Reply:
widget.title seems like a good call here.